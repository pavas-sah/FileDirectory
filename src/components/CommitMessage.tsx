import React from 'react';

export interface CommitMessageProps {
  message: string;
}

export const CommitMessage: React.SFC<CommitMessageProps> = (props) => {
  return <>{props.message}</>;
};
export default CommitMessage;
