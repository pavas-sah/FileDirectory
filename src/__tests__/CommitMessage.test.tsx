import React from 'react';
import CommitMessage from '../components/CommitMessage';
import renderer from 'react-test-renderer';

const MessageProps = {
  message: 'some message',
};

describe('Commit Message Functional component', () => {
  it('should have some message as content.', () => {
    const messageComponent = renderer.create(
      <CommitMessage {...MessageProps} />
    );
    expect(messageComponent.root.children.toString()).toBe('some message');
  });

  it('matches snapshot', () => {
    const tree = renderer.create(<CommitMessage {...MessageProps} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
