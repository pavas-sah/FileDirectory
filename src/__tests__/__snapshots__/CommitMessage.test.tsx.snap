// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Commit Message Functional component matches snapshot 1`] = `"some message"`;
